### ISSUE TITLE
Vulkan device lost on Intel Graphics 605

### ISSUE THREAD

**Issue by qazxsw4320** [hrydgard/ppsspp#17528]
### Game or games this happens in

God of war chains of Olympus, star wars the force unleashed, dragon ball z tenkaichi tag team, and god of war ghost of sparta, Gun showdown

### What area of the game

Right when I start the game in both Chains of Olympus and ghost of sparta it gives me the vulkan lost error. In the force unleashed right when I get to the title screen is causes this and in the dragon ball z tenkaichi tag team whenever I try to change resolution is causes the vulkan to be lost.
Finally with gun showdown right after the first cutscene at the beginning of the game the vulkan device is lost.
### What happens

![Screenshot 2023-05-20 110012](https://github.com/hrydgard/ppsspp/assets/122139385/1a44ccbc-140c-44e0-8fd8-7059084bc39a)
Vulkan device gets lost

### What should happen

It should work. I did address the problem about 2 weeks ago and was told to update my drivers and I did. It resolved the issue that one time but now when I try to play it again the error happens again. It's not all games that this happens. The vast majority of games don't have the issue. It's only the ones I listed that this happens. I tried to update my drivers again and even tried it on another laptop I have and the issue still remains. Other backends like direct3d11 and opengl work fine it's just the vulkan backend where this happens.

### Platform

Windows

### Mobile phone model or graphics card

UHD intel graphics 605

### PPSSPP version affected

1.15.4

### Last working version

1.14.4

### Graphics backend (3D API)

Vulkan

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try changing graphics settings to determine if one causes the glitch (especially speed hacks or enhancements/replacements.)
- [X] Include logs or screenshots of issue.

**Comment by hrydgard** [comment]
Is this still an issue? Haven't heard of any more instances of this.

**Comment by qazxsw4320** [comment]
@hrydgard yes, but I messed up and said graphics 605. It's graphics 600. I update my drivers and it still crashes sometimes.

**Comment by hrydgard** [comment]
Right, but have you tried a new build of PPSSPP? Like the latest from https://buildbot.orphis.net/ppsspp/ .

**Comment by qazxsw4320** [comment]
@hrydgard It's everything since version 15.0 came out. Here is the latest build and it still happens. For example, this is dbz tenkaichi tag team and this happens when I change resolution. But another example is the force unleashed. Right when I get into the main menu the same error pops up.
![20231214_205303](https://github.com/hrydgard/ppsspp/assets/122139385/98079a32-6be9-4d90-a644-c7a3cfe0b835)
